### ISSUE TITLE
Exception being thrown in tasks.py

### ISSUE THREAD

**Issue by jblaine** [okorach/sonar-tools#871]
Using sonar-tools 2.6, and during a run of sonar-housekeeper --dry-run, I get the following among the other output:
```
Exception in thread ProjectAudit0:
Traceback (most recent call last):
  File "/home/jblaine/python/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/home/jblaine/python/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/home/jblaine/testenv/lib/python3.9/site-packages/sonar/projects/projects.py", line 1100, in __audit_thread
    results += project.audit(audit_settings)
  File "/home/jblaine/testenv/lib/python3.9/site-packages/sonar/projects/projects.py", line 511, in audit
    + self._audit_bg_task(audit_settings)
  File "/home/jblaine/testenv/lib/python3.9/site-packages/sonar/components.py", line 179, in _audit_bg_task
    return last_task.audit(audit_settings)
  File "/home/jblaine/testenv/lib/python3.9/site-packages/sonar/tasks.py", line 440, in audit
    problems += self.__audit_scanner_version(audit_settings)
  File "/home/jblaine/testenv/lib/python3.9/site-packages/sonar/tasks.py", line 397, in __audit_scanner_version
    release_date = SCANNER_VERSIONS[scanner_type][str_version]
KeyError: '4.8.0'
```

**Comment by okorach** [comment]
Hello @jblaine,

Thanks for reporting this. The problem is obvious (and that could be the same root problem that affects #872).
This is due to a new version of the scanner that was used to analyze the problem.

I will fix it ASAP

**Comment by okorach** [comment]
Problem probably due to #874

**Comment by okorach** [comment]
Should be fixed with the just released sonar-tools 2.7
